src/influxdb.c

```diff
@@ -111,6 +112,30 @@ corto_int16 influxdb_serScalar(
     return 0;
 }
 
+corto_int16 influxdb_serComposite(
```

Comment:
Suggest to rename this to `influxdb_serComplex` ('complex' is a convention used elsewhere to describe composite + collection)